src/CommandLineParser.ts

```diff
@@ -162,6 +172,39 @@ function inputConflicts(videoUrls: Array<string | number> | undefined,
 }
 
 
+function isOutputTemplateValid(argv: any): boolean {
+    let finalTemplate: string = argv.outputTemplate;
+    const elementRegEx = RegExp(/{(.*?)}/g);
+    let match = elementRegEx.exec(finalTemplate);
+
+    // if no template elements this fails
+    if (match) {
+        // keep iterating untill we find no more elements
+        while (match) {
+            if (!templateElements.includes(match[1])) {
+                logger.error(
+                    `'${match[0]}' is not aviable as a template element \n` +
+                    'Please check the template elements aviable in the README \n',
```

Comment:
This is fine, however i'd say print all the elements we support right there in the error message rather than pointing users to the README.

Reply:
K, all fixes will be done 12hrs from now at least : D

Reply:
There is no deadline in opensource because
![](https://webdevstudios.com/wp-content/uploads/2015/03/its-free-software.gif)

Reply:
✅